### ISSUE TITLE
Unable to match a single `\` character

### ISSUE THREAD

**Issue by unvariant** [yoav-lavi/melody#91]
**Describe the bug**
Unable to write melody that generates correct regex that will match a single `\`.

**To Reproduce**
test.mldy
```
<start>;
"\";
<end>;
```
```
melody test.mldy
Error:  --> 2:1
  |
2 | "\";
  | ^---
  |
  = expected EOI, literal, raw, not, range, quantifier_quantity, group_declaration, assertion_declaration, variable_declaration, or variable_invocation
```

**Expected behavior**
Expected melody to properly escape `\`.

**Desktop (please complete the following information):**
 - OS: macOS
 - Version: 12.4

**Additional context**
I tried some other methods to generate regex that matches `\`, but none of them worked.
```
"\\";
```
compiles fine but escapes both slashes, generating a regex that matches `\\` instead of `\`.
I also tried using the raw syntax (none of them worked):
```
`\`;
```
```
`\\`;
```
```
`[\\]`;
```

**Comment by yoav-lavi** [comment]
Thanks for the report!
Will look into this

**Comment by yoav-lavi** [comment]
This is a bit of a tricky one, I'll take a deeper look into it but for now I'll add a `<backslash>` symbol to allow you to use this. Will be out in the next version

**Comment by yoav-lavi** [comment]
Will be fixed in 0.19.0

**Comment by yoav-lavi** [comment]
Note that `"\\";` will be the correct syntax
